docs/releases/v2016.2.4.rst

```diff
@@ -0,0 +1,54 @@
+Gluon 2016.2.4
+==============
+
+Bugfixes
+~~~~~~~~
+
+* Fix batman-adv not being able to transmit packages of specific sizes (`b7eeef9 <https://github.com/freifunk-gluon/gluon/commit/b7eeef9b04b44a70b2a953c4efe35a3fdceba2db>`_)
+
+  We suspect that this issue was also the reason for the autoupdater/wget hangs observed by many communities.
+
+* Fix build after ftp.all.kernel.org discontinuation (`#1059 <https://github.com/freifunk-gluon/gluon/issues/1059>`_)
+
+* Fix high load because of frequent calls of the respondd initscript (`9a0aeb9 <https://github.com/freifunk-gluon/gluon/commit/9a0aeb9b7482df4e4515e61356b9d393e3a7eacb>`_)
+
+  The respondd restart triggers added in v2016.2.3 ran a significant portion of the respondd initscript for each router advertisement
+  received. This was fixed by a backport of a netifd patch.
+
+* x86 sysupgrade fixes (`41fd50d <https://github.com/freifunk-gluon/gluon/commit/41fd50d20ba31d73c4796c5b2d4eb44ad2258b90>`_,
+  `ad37e2b <https://github.com/freifunk-gluon/gluon/commit/ad37e2b6b43b2c3389356d892b04f3873d8f6b93>`_)
+
+  This fixes sysupgrade on mmcblk and similar devices.
+
+Other changes
+~~~~~~~~~~~~~
+
+* The manifest generator has been extended to generate SHA256 checksums in addition to SHA512 ones
+  (`f9d59be <https://github.com/freifunk-gluon/gluon/commit/f9d59be731efd31a26c59e049ccbdc4b1762f6b1>`_)
+
+  We have recently switched the autoupdater to SHA256 to avoid mixing two different lengths of hashes for
```

Comment:
You might want to mention "the autoupdater on master" or "for 2017.1" or whatever, because the autoupdater in 2016.2.4 *doesn't* check for SHA256 checksums.